<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>window</class>
 <widget class="QMainWindow" name="window">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>594</width>
    <height>561</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Search Here</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: #343434;
</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="heading_label">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>20</y>
      <width>231</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Georgia</family>
      <pointsize>28</pointsize>
      <weight>75</weight>
      <italic>true</italic>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel
{
	color: tomato;
}</string>
    </property>
    <property name="text">
     <string>Search App</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="search_entry">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>110</y>
      <width>311</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Georgia</family>
      <pointsize>18</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QLineEdit
{
		border: 2px solid aqua;
	 	color: limegreen
}
</string>
    </property>
    <property name="placeholderText">
     <string>Search here</string>
    </property>
   </widget>
   <widget class="QPushButton" name="search_button">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>110</y>
      <width>161</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Lucida Console</family>
      <pointsize>18</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="toolTip">
     <string>Search information</string>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton
{
	color: cyan
}</string>
    </property>
    <property name="text">
     <string>&amp;Search</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="output_area">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>190</y>
      <width>511</width>
      <height>321</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Georgia</family>
      <pointsize>18</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QTextEdit
{
	background-color: #343434;
	border: 2px solid aqua;
	color: hotpink;
}</string>
    </property>
    <property name="acceptRichText">
     <bool>true</bool>
    </property>
    <property name="placeholderText">
     <string>Output will appear here</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>594</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
